DuckDuckGo/OnboardingExperiment/Pixels/OnboardingPixelReporterStore.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  OnboardingPixelReporterStore.swift
+//  DuckDuckGo
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import Core
+
+protocol OnboardingPixelReporterStorage: AnyObject {
+    var hasFiredIntroScreenShownPixel: Bool { get set }
+    var hasFiredComparisonChartShownPixel: Bool { get set }
+}
+
+final class OnboardingPixelReporterStore: OnboardingPixelReporterStorage {
```

Comment:
I don’t think we need an explicit store… There is already a UniquePixel class that does everything for you already.
The only caveat is that we need to allow the pixel suffix to be _unique as well as _u.